elytron/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemTransformers.java

```diff
@@ -142,10 +145,17 @@ public void registerTransformers(SubsystemTransformerRegistration registration)
         // 2.0.0 (WildFly 12) to 1.2.0, (WildFly 11 and EAP 7.1.0)
         from2(chainedBuilder);
 
-        chainedBuilder.buildAndRegister(registration, new ModelVersion[] { ELYTRON_17_0_0, ELYTRON_16_0_0, ELYTRON_15_1_0, ELYTRON_15_0_0, ELYTRON_14_0_0, ELYTRON_13_0_0, ELYTRON_12_0_0, ELYTRON_11_0_0, ELYTRON_10_0_0, ELYTRON_9_0_0,
+        chainedBuilder.buildAndRegister(registration, new ModelVersion[] { ELYTRON_18_0_0, ELYTRON_17_0_0, ELYTRON_16_0_0, ELYTRON_15_1_0, ELYTRON_15_0_0, ELYTRON_14_0_0, ELYTRON_13_0_0, ELYTRON_12_0_0, ELYTRON_11_0_0, ELYTRON_10_0_0, ELYTRON_9_0_0,
                 ELYTRON_8_0_0, ELYTRON_7_0_0, ELYTRON_6_0_0, ELYTRON_5_0_0, ELYTRON_4_0_0, ELYTRON_3_0_0, ELYTRON_2_0_0, ELYTRON_1_2_0 });
     }
 
+    private static void from19(ChainedTransformationDescriptionBuilder chainedBuilder) {
```

Comment:
@pferraro Following from your comment [here](https://github.com/wildfly/wildfly-core/pull/4311#discussion_r1534498261), I have added a new version 19 of the elytron subsystem, and added schemas for both the DEFAULT and the COMMUNITY stability for this version.  I also added a transformer from the new version 19 to version 18, that you can see here. 

I have ignored the stability levels when I added this transformer. And I did not add any transformers between different stability levels of the same version. Just wanted to double check that this is correct, please? Thank you!

Reply:
Since the transformer is rejection only, this should function correctly for either stability level (rejecting a child resource with no registration should just be no-op).
Additionally, stability level of the target host is not yet exposed via SubsystemTransformerRegistration - so this would not yet be possible anyway.